templates/admin-section.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ *
+ * @author
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+style('files_external_beeswarm', 'settings-admin');
+script('files_external_beeswarm', 'settings');
+?>
+<div id="theming" class="section">
+	<h2 class="inlineblock"><?php p($l->t('Bee Swarm Storage')); ?></h2>
+	<a target="_blank" rel="noreferrer" class="icon-info" title="<?php p($l->t('Open documentation'));?>" href="<?php p(link_to_docs('admin-beeswarm')); ?>"></a>
```

Comment:
I don't  believe we have any docs?

Reply:
Not yet, but I'm assuming we will

Reply:
Most likely, but this I believe is a Nextcloud specific function that links to the docs shipped with Nextcloud. So we're most likely going to do it differently, so we can remove it now and then look at adding it when we have some docs.

Reply:
Agreed.

Reply:
Still hasn't been removed, alternatively you can switch it to go to the Github page, since I think that might be a decent place to put the wiki.

Reply:
There's probably a way to read the website tag from the info.xml - but haven't found a way yet to do it.
For the moment it's hardcoded to the url.